authorization/api-secret-key/README.md

```diff
@@ -0,0 +1,62 @@
+# Tapsi Pack API: API Secret Key Guide
+
+Welcome to the API Secret Key guide for the Tapsi Pack API. This guide will help you understand how to authenticate with the
+Tapsi Pack API using API Secret Key
+
+## Introduction
+In this document, external teams integrating with our API are referred to as **clients**.
```

Comment:
Same here there are no clients, also I guess we should use it as singular.

Reply:
the plural form is not wrong here, also We should define clients here too because the clients are used in this documentation and it's needed